src/savers/tvg/tvgTvgSaver.cpp

```diff
@@ -412,7 +412,7 @@ TvgBinCounter TvgSaver::serializeComposite(const Paint* cmpTarget, CompositeMeth
     auto flag = static_cast<TvgBinFlag>(cmpMethod);
     auto cnt = writeTagProperty(TVG_TAG_PAINT_CMP_METHOD, SIZE(TvgBinFlag), &flag);
 
-    cnt += serialize(cmpTarget, nullptr);
+    cnt += serialize(cmpTarget, nullptr, true);
```

Comment:
@hermet I just wanted to approve :)
I wondered, whether it is worth to check, whether the compTarget is the InvAlphaMask and if so call  ```cnt += serialize(cmpTarget, nullptr);``` but I don't think it's a very common case and we'd need to add an additional if